dataactbroker/userRoutes.py

```diff
@@ -13,22 +17,26 @@ def add_user_routes(app,system_email,bcrypt):
     """
 
     @app.route("/v1/register/", methods = ["POST"])
-    #check the session to make sure register is set to prevent any one from using route
-    @permissions_check(permission="check_email_token")
     def register_user():
         """ Expects request to have keys 'email', 'name', 'cgac_code', and 'title' """
-        accountManager = AccountHandler(request,bcrypt = bcrypt)
-        return accountManager.register(system_email, session)
+        # check the session to make sure register is set to prevent any one
+        # from using route
+        if LoginSession.isRegistering(session):
+            accountManager = AccountHandler(request,bcrypt = bcrypt)
+            return accountManager.register(system_email, session)
+        else:
+            return JsonResponse.create(StatusCode.LOGIN_REQUIRED,
+                                       {'message': 'unauthorized'})
```

Comment:
In terms of this function/route and the `update_user` and `delete_user` routes (and whatever other ones fall under this category), do we need to be updating them at all? I thought we were going to remove them entirely (can we do that yet?)